### PR TITLE
Added maintainers to spec

### PR DESCRIPTION
**Pull request by RichardLitt**
See #30

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -157,6 +157,18 @@ progressively greater amount of knowledge."
...
 
+### Maintainer(s)
+**Status**: Required.
+
+**Requirements:**
+- Must be called `Maintainer` or `Maintainers`.
+- List maintainer(s) for a repository, along with one way of contacting them (e.g. GitHub link or email).
+
+**Suggestions:**
+- This should be a small list of people in charge of the repo. This should not be everyone with access rights, such as an entire organization, but the people who should be pinged and who are in charge of merging PRs and so on.
```
Comment (VictorBjelkholm) [inline spec.md]: Feels like it's missing a definition of "in charge". Currently it only mentions that it's people who are "in charge of merging PRs"

File: spec.md [hunk 2]
```diff
@@ -157,6 +157,18 @@ progressively greater amount of knowledge."
...
+- A subsection for contributors is also welcome here, but can go in the contribute section, too.
```
Comment (VictorBjelkholm) [inline spec.md]: Since we're trying to build a standard for readmes, it would be better I think if we only have one way of doing things instead.

**Comment by RichardLitt** [comment]
@VictorBjelkholm Do you get notifications if I force push? Curious.

**Comment by VictorBjelkholm** [comment]
@RichardLitt no, don't think so. Just get notifications regarding the mentions. I do think this looks good now 👍

**Comment by RichardLitt** [comment]
Ok, will remember to ping. @martinheidegger, thoughts?

**Pull request merged**